templates/account-signin.php

```diff
@@ -36,20 +36,20 @@
 	$use_standard_password_email = wpjm_use_standard_password_setup_email();
 	?>
 	<fieldset class="fieldset-login_required">
-		<label><?php _e( 'Have an account?', 'wp-job-manager' ); ?></label>
+		<label><?php esc_html_e( 'Have an account?', 'wp-job-manager' ); ?></label>
 		<div class="field account-sign-in">
-			<a class="button" href="<?php echo apply_filters( 'submit_job_form_login_url', wp_login_url( get_permalink() ) ); ?>"><?php _e( 'Sign in', 'wp-job-manager' ); ?></a>
+			<a class="button" href="<?php echo esc_url( apply_filters( 'submit_job_form_login_url', wp_login_url( get_permalink() ) ) ); ?>"><?php esc_html_e( 'Sign in', 'wp-job-manager' ); ?></a>
 
 			<?php if ( $registration_enabled ) : ?>
 
-				<?php printf( __( 'If you don\'t have an account you can %screate one below by entering your email address/username.', 'wp-job-manager' ), $account_required ? '' : __( 'optionally', 'wp-job-manager' ) . ' ' ); ?>
+				<?php printf( esc_html__( 'If you don\'t have an account you can %screate one below by entering your email address/username.', 'wp-job-manager' ), $account_required ? '' : esc_html__( 'optionally', 'wp-job-manager' ) . ' ' ); ?>
 				<?php if ( $use_standard_password_email ) : ?>
-					<?php printf( __( 'Your account details will be confirmed via email.', 'wp-job-manager' ) ); ?>
+					<?php printf( esc_html__( 'Your account details will be confirmed via email.', 'wp-job-manager' ) ); ?>
 				<?php endif; ?>
 
 			<?php elseif ( $account_required ) : ?>
 
-				<?php echo apply_filters( 'submit_job_form_login_required_message',  __('You must sign in to create a new listing.', 'wp-job-manager' ) ); ?>
+				<?php echo wp_kses_post( apply_filters( 'submit_job_form_login_required_message',  __('You must sign in to create a new listing.', 'wp-job-manager' ) ) ); ?>
```

Comment:
This was an original lint issue, but while editing line, there is a spacing issue in the argument list for `__()`.

Reply:
fixed